examples/rbac/clusterrole/services.yaml

```diff
@@ -0,0 +1,10 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+metadata:
+  name: services
```

Comment:
these are superuser permissions... I wouldn't use them as an example of the permissions to grant all service accounts. https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/controller-roles.yaml has roles defined for all the core controllers, and https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/cluster-roles.yaml has roles for superuser (`cluster-admin`) and namespace-scoped users (`admin`, `edit`, and `view`) which would be good starting points for something you might grant a service account.